### ISSUE TITLE
Dataset update

### ISSUE THREAD

**Issue by mimbres** [mimbres/neural-audio-fp#4]
**Dataset update** 
- **Early downloaders (before July 8, 2021, 2:31 PM GMT)** need this update via [Dataport](http://ieee-dataport.org/4455).
- The test set included in the initial package was labeld as SNR [0,10] dB,  but it was actually [10,10] dB (easier test). Mistake during directory cleanup. It's fixed now. 
- Now v1.1 with SNR= [0,10]dB, 0dB, -3dB queries.
- I found 24 duplicate (between the dummy and test sets) songs in the previous data set for publication results. I also had to replace a few songs for the training set. Datasets v1-v1.1 is very clean, as I double-checked. 
- One problem (?) is that after the data set correction, the performance improved by almost 5 to 6 percent for 1 second query.

In progress:
- [x] Fast download with command-line interface (kaggle public dataset?)
- [x] md5/sha1 checksum
- [x] Upload full dataset after correcting dataset duplicates
